graph.go

```diff
@@ -343,9 +341,13 @@ func doGraph(pkgs []*build.Package) (*gog.Output, error) {
 		pkgInfos = append(pkgInfos, pkg)
 	}
 
+	var output gog.Output
 	for _, pkg := range pkgInfos {
-		g.Graph(prog.Fset, pkg.Files, pkg.Pkg, &pkg.Info, true)
+		o := gog.Graph(prog.Fset, pkg.Files, pkg.Pkg, &pkg.Info, true)
+		output.Defs = append(output.Defs, o.Defs...)
+		output.Refs = append(output.Refs, o.Refs...)
+		output.Docs = append(output.Docs, o.Docs...)
```

Comment:
maybe `gog.Output` would benefit from a `Merge` or `Append` function so we don't repeat these lines all over the place.